src/pages/Home.vue

```diff
@@ -38,11 +45,69 @@ export default {
       default: "",
       type: String,
     },
+    program: {
+      /** The program the user falls under. Eg: HaryanaStudents, DelhiStudents */
+      default: "",
+      type: String,
+    },
+    authType: {
+      /** The authentication method used by the user */
+      default: "",
+      type: String,
+    },
+  },
+  data() {
+    return {
+      textObject: {
+        placeholderText: "",
+        displayText: "",
+        addButtonText: "",
+        invalidInputMessage: "",
+        invalidLoginMessage: "",
+        submitButtonText: "",
+      },
+      maxNumberOfInput: 0,
+      inputObject: {
+        basicInputValidationType: "",
+        maxLengthOfId: 0,
+        mode: null,
+        type: null,
+      },
+      dataSourceObject: {
+        name: "",
+        column: "",
+      },
+    };
   },
   computed: {
+    /** Whether multiple ID can be entered */
     isMultipleIDEntryAllowed() {
       return this.redirectTo == "plio";
     },
   },
+  async created() {
+    /** Program name is sent to the API to retrieve all details */
+    var programData = await firebaseAPI.getProgramData(this.program);
+
+    let textObject = programData["text"]["default"];
```

Comment:
What is this `default`? In the JSON doc on notion I could see no `default`?
![image](https://user-images.githubusercontent.com/20169829/146392830-5245cea5-56a0-4b7c-ba9b-e5fed94d485d.png)